config/etc/system-image/writable-paths

```diff
@@ -13,6 +13,7 @@
 /tmp                                    none                    temporary   none        defaults
 /mnt                                    none                    temporary   none        defaults
 /media                                  none                    temporary   none        defaults
+/boot/androidboot                       auto                    persistent  none        none
```

Comment:
We would need a partition different from bootloader/boot/recovery/system, and I am not sure we have that. And I'd say all things you mention need to be modified anyway when adding a new bootloader.